.gitignore

```diff
@@ -15,13 +16,22 @@ config*.log
 /native/**/*.a
 /native/**/*.so
 /native/**/*.vcxproj.user
+/native/**/*.aps
 /native/**/Win32/
 /native/**/x64/
+/native/**/Release
+/native/exe/HCWIRBlasterCOM/HCWIRBlasterCOM_*
 
+/third_party/ffmpeg/lib*/*_list.c
+/third_party/ffmpeg/lib*/*.ver 
+/third_party/ffmpeg/doc 
 /third_party/**/*.o
 /third_party/**/*.lo
+/third_party/**/*.a
```

Comment:
Don't add this one, this refers to existing files in GIT

Reply:
Generally .a shouldn't be in a source repo but I see the .a files under mingw/dx80/lib only.  Is it ok if I limit it to .a files created by compiling ffmpeg?

Reply:
Yeah, I know, but we had to include some binaries in the repo for compatibility reasons. If you limit it to ffmpeg then that's fine.